experimental/uefi/loader/src/main.rs

```diff
@@ -79,16 +80,29 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         }
     });
 
+    // Unfortunately OVMF writes some garbage (clear screen etc?) + our Hello World to the
+    // other serial port, so let's skip some bytes before we set up the CBOR codec.
+    let mut junk = [0; 71];
```

Comment:
Is there some way that we can tell the junk is finished? Hardcoding a fixed number is brittle. I think it is fine for now, as the serial port is only a temporary mechanism, but perhaps make the 71 a constant with a more descriptive name.

Reply:
Yeah, this is a massive hack. I'll file a bug and add a proper TODO about this.

A much cleaner way would be to figure out how to configure OVMF not to output that junk to the second serial port (ideally it'd be only the Hello World message that gets sent to the first serial port as well), but I haven't yet found if it's possible to do this.

Interestingly, log messages get only sent to the first serial port, not the second. Maybe it's something that stdout gets sent to all serial ports by default, and stderr gets sent to the first, or something.